packages/opentelemetry/src/custom/transaction.ts

```diff
@@ -23,29 +22,3 @@ export function startTransaction(hub: HubInterface, transactionContext: Transact
   }
   return transaction;
 }
-
-/**
- * This is a fork of the base Transaction with OTEL specific stuff added.
- */
-export class OpenTelemetryTransaction extends Transaction {
```

Comment:
not needed anymore - we just leverage this in the core implementation through `getCapturedScopesOnSpan`!